src/notification/notification.service.ts

```diff
@@ -148,6 +155,16 @@ export class NotificationService {
                     options.accountId.toString(),
                 ).andOn('post_reposts.post_id', 'post.id');
             })
+            .leftJoin('follows as follows_actor', function () {
+                this.onVal(
```

Comment:
This won't work. I tested this for notifications and `actor_followed_by_user` will always be `0` -  When using `onVal`, `'actor_account.id'` will be treated as a string. 

We should use:-

```
this.on(
                    'follows_actor.following_id',
                    'actor_account.id',
                )
```

Reply:
I also think we should add some tests that check for `"actor_followed_by_user": 1,` so that we don't miss this case.